cluster-setup/ci-cluster/feature_ptp/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../base/feature_ptp
+patches:
+  - subscription.yaml
```

Comment:
Why this patch? The target should be 4.4

Reply:
here we use the test source.

Production: `redhat-operators`
Test: `opsrctest` the should point to the ART source catalog